crates/holochain/src/sweettest/sweet_app_installation.rs

```diff
@@ -67,9 +67,17 @@ pub async fn get_install_app_payload_from_dnas(
 ) -> InstallAppPayload {
     let dnas_with_roles: Vec<_> = data.iter().map(|(dr, _)| dr).cloned().collect();
     let bundle = app_bundle_from_dnas(&dnas_with_roles, false, network_seed).await;
-    let membrane_proofs = Some(
+    let roles_settings = Some(
         data.iter()
-            .map(|(dr, memproof)| (dr.role(), memproof.clone().unwrap_or_default()))
+            .map(|(dr, memproof)| {
+                (
+                    dr.role(),
+                    RoleSettings::Provisioned {
```

Comment:
Role settings implements default, just `Default::default()` here?

Reply:
Default would omit the membrane proof but we need it here.